ChallengePart1.R

```diff
@@ -0,0 +1,66 @@
+rm(list=ls())

```

Comment:
I'm confused if I should be looking at this or the other scripts? Make sure to keep your answers in the same file and trade that back and forth instead of creating separate files. Version control is meant to keep track of the same set of files not multiple sets of files doing the same thing. Does that make sense?